src/app/components/Home/Home.jsx

```diff
@@ -34,6 +34,7 @@ class Home extends React.Component {
   render() {
     return (
       <DocumentTitle title="Shared Collection Catalog | NYPL">
+      <main>
```

Comment:
Looks good. Just indent everything by one since it's a child of `DocumentTitle`.